@QueryDelegate is not supported on methods of static inner classes

Here is just a minor bug I stumbled upon...

When placing the ```@QueryDelegate``` annotation on a static method of a static inner class, the resulting Q-class can not be compiled since it does not qualify the delegate's classname correctly.

Here is an example:
```java
package demo;

import java.util.Date;

import com.mysema.commons.lang.Pair;
import com.mysema.query.annotations.QueryDelegate;
import com.mysema.query.types.expr.BooleanExpression;
import com.mysema.query.types.path.DateTimePath;

public class QueryExtensions {

  public static class DateExtensions {
    @QueryDelegate(Date.class)
    public static BooleanExpression inPeriod(DateTimePath<Date> date, Pair<Date, Date> period) {
      return date.goe(period.getFirst()).and(date.loe(period.getSecond()));
    }
  }
}
```

This is the generated class:
```java
package ext.java.util;

import static com.mysema.query.types.PathMetadataFactory.*;
import java.util.Date;

import demo.*;

import com.mysema.query.types.path.*;

import com.mysema.query.types.PathMetadata;
import javax.annotation.Generated;
import com.mysema.query.types.Path;

@Generated("com.mysema.query.codegen.EmbeddableSerializer")
public class QDate extends DateTimePath<Date> {

    private static final long serialVersionUID = 65575278L;

    public static final QDate date = new QDate("date");

    public QDate(String variable) {
        super(Date.class, forVariable(variable));
    }

    public QDate(Path<? extends Date> path) {
        super(path.getType(), path.getMetadata());
    }

    public QDate(PathMetadata<?> metadata) {
        super(Date.class, metadata);
    }

    public com.mysema.query.types.expr.BooleanExpression inPeriod(com.mysema.commons.lang.Pair<Date, Date> period) {
        return DateExtensions.inPeriod(this, period);
    }

}
```
The compiler issues the following message:

```
/pathtoproject/build/classes/main/ext/java/util/QDate.java:38: error: cannot find symbol
        return DateExtensions.inPeriod(this, period);
               ^
  symbol:   variable DateExtensions
  location: class QDate
```

--- a/querydsl-apt/src/test/apt/com/querydsl/InnerExtensions.java
+++ b/querydsl-apt/src/test/apt/com/querydsl/InnerExtensions.java
@@ -1,0 +1,15 @@
+package com.querydsl;
+
+import com.querydsl.core.annotations.QueryDelegate;
+import com.querydsl.core.types.dsl.BooleanExpression;
+
+public class InnerExtensions {
+
+    public static class ExampleEntity2Extensions {
+
+        @QueryDelegate(ExampleEntity2.class)
+        public static BooleanExpression isZero(QExampleEntity2 left) {
+            return left.id.eq(0);
+        }
+    }
+}

--- a/querydsl-apt/src/test/java/com/querydsl/apt/domain/InnerExtensionsTest.java
+++ b/querydsl-apt/src/test/java/com/querydsl/apt/domain/InnerExtensionsTest.java
@@ -1,0 +1,30 @@
+package com.querydsl.apt.domain;
+
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+import com.google.common.base.Charsets;
+import com.google.common.io.Files;
+import com.querydsl.apt.AbstractProcessorTest;
+import com.querydsl.apt.QuerydslAnnotationProcessor;
+
+public class InnerExtensionsTest extends AbstractProcessorTest {
+
+    private static final String packagePath = "src/test/apt/com/querydsl/";
+
+    @Test
+    public void Process() throws IOException {
+        List<String> sources = Arrays.asList(
+                new File(packagePath, "InnerExtensions.java").getPath(),
+                new File(packagePath, "ExampleEntity2.java").getPath());
+        process(QuerydslAnnotationProcessor.class, sources, "innerextensions");
+        String qtypeContent = Files.toString(new File("target/innerextensions/com/querydsl/QExampleEntity2.java"), Charsets.UTF_8);
+        assertTrue(qtypeContent.contains("return InnerExtensions.ExampleEntity2Extensions.isZero(this);"));
+    }
+}

--- a/querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java
+++ b/querydsl-codegen/src/main/java/com/querydsl/codegen/EntitySerializer.java
@@ -563,7 +563,7 @@ public class EntitySerializer implements Serializer {
         writer.beginPublicMethod(delegate.getReturnType(), delegate.getName(), params);
 
         // body start
-        writer.beginLine(RETURN + delegate.getDelegateType().getSimpleName() + "." + delegate.getName() + "(");
+        writer.beginLine(RETURN + writer.getRawName(delegate.getDelegateType()) + "." + delegate.getName() + "(");
         writer.append("this");
         if (!model.equals(delegate.getDeclaringType())) {
             int counter = 0;

